rskj-core/src/main/java/org/ethereum/rpc/JsonRpcNettyServer.java

```diff
@@ -65,7 +75,7 @@ protected void initChannel(SocketChannel ch) throws Exception {
                     p.addLast(jsonRpcWeb3ServerHandler);
                 }
             });
-        b.bind(port).sync();
+        b.bind(host, port).sync();
```

Comment:
Note that this line is the one where we stop binding to `0.0.0.0` and bind to a specific host instead. By default, we'll bind to the loopback address now :clap:.